app.test.js

```diff
@@ -0,0 +1,41 @@
+const { app } = require("./app");
+const request = require("supertest");
+const connection = require("./db/connection");
+const { seed } = require("./db/seeds/seed");
+const data = require("./db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return connection.end();
+});
+
+describe("/api/topics", () => {
+  describe("GET topics default", () => {
+    test("200: it responds with an array of topics", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          const { topics } = response.body;
+          expect(Array.isArray(topics)).toBe(true);
+        });
+    });
+
+    test("200: each topic object should have a slug and description property", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then((response) => {
+          const { topics } = response.body;
+
```

Comment:
Great work using a forEach! As we're using this though, could you just add another line that checks the length of the array beforehand? 🙂